api/desecapi/management/commands/chores.py

```diff
@@ -46,7 +48,44 @@ def update_healthcheck_timestamp():
             serializer.save()
         print(f'TXT {name} updated to {content}')
 
+    @staticmethod
+    def alerting_healthcheck():
+        name = 'external-timestamp.desec.test'
+        try:
+            models.Domain.objects.get(name=name)
+        except models.Domain.DoesNotExist:
+            print(f'{name} zone is not configured; skipping alerting health check')
+            return
+
+        now = time.time()
+        timestamps = []
+        query = dns.message.make_query(f'{name}.', dns.rdatatype.TXT)
+        server = 'ns1.desec.io'
+        response = None
+        try:
+            response = dns.query.tcp(query, server, timeout=5)
+            for rrset in response.answer:
+                if rrset.rdtype != dns.rdatatype.TXT:
+                    continue
+                for txt_content in rrset:
+                    timestamps.append(str(txt_content)[1:-1])
+
+            if any(now - 600 <= int(timestamp) <= now for timestamp in timestamps):
+                print(f'TXT {name} up to date.')
+                return
+        except Exception:
+            pass
```

Comment:
indent the following block to make the structure more clear

Reply:
No, the following block should also execute when there was no exception

Reply:
improved structural clarity some other way